confetti/src/main/java/com/github/jinatonic/confetti/confetto/Confetto.java

```diff
@@ -326,10 +339,12 @@ private float computeDistance(long t, float xi, float vi, float ai, Long targetT
      */
     public void draw(Canvas canvas) {
         if (touchOverride) {
-            draw(canvas, overrideX + overrideDeltaX, overrideY + overrideDeltaY, currentRotation,
-                    percentageAnimated);
+            velocityTracker.computeCurrentVelocity(1);
```

Comment:
not a fan of using velocityTracker to recompute velocity on every draw after the user touches the confetto, it can be a potentially expensive process. the point of velocity tracker is to track the user's touch and override the initial drawing states to take the user's motion into account. the code inside `computeDistance` SHOULD just work since it uses the overridden state variables, but please double check to make sure.